browser/src/view/components/Button/Button.scss

```diff
@@ -1,26 +1,37 @@
 @use '../../../styles/common/variables' as *;
 
-.btn-icon-container {
-  width: 48px;
+.btn-primary {
   height: 48px;
+  width: 48px;
```

Comment:
We should probably set a body font size default and use rems for sizing so screen resizes are treated properly.

Reply:
Def. I think body font size of 16px is one common practice iirc so I will explicitly set it to that.